lib/init.coffee

```diff
@@ -1,6 +1,11 @@
+os = require 'os'
```

Comment:
...for some reason I thought you had already done this when I wrote that comment. Ignore this! :stuck_out_tongue: